spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,22 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+Trailing commas are allowed when followed by a newline and a closing
```

Comment:
My suggestion is to remove the ambiguity in the plural "commas" and what it means for a comma to be "allowed".